### PR TITLE
doc/cephadm: add section about setting config settings

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Fixes: https://tracker.ceph.com/issues/44284

https://lists.ceph.io/hyperkitty/list/ceph-users@ceph.io/thread/EE2EKUR5DJGA5EHDEEBIRQWE632XMNS2/

Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephadm/operations.rst [hunk 1]
```diff
@@ -254,3 +254,24 @@ You can remove a broken host from management with::
 You can disable this health warning with::
 
   ceph config set mgr mgr/cephadm/warn_on_failed_host_check false
+
+/etc/ceph/ceph.conf
+===================
+
+Cephadm only uses a minimal ``ceph.conf`` only containing the 
+minimal set of information to connect to the Cep cluster.
```
Comment (mgfritch) [inline doc/cephadm/operations.rst:262]:
```suggestion
Cephadm uses a minimal ``ceph.conf`` only containing the 
minimal set of information to connect to the Ceph cluster.
```

File: doc/cephadm/operations.rst [hunk 2]
```diff
@@ -254,3 +254,24 @@ You can remove a broken host from management with::
...
+
+In order to set configuration settings, please use::
+
+  ceph config set ...
+
+to update the configuration.
```
Comment (mgfritch) [inline doc/cephadm/operations.rst:268]:
```suggestion
To update the configuration settings, please use::

  ceph config set ...

```

File: doc/cephadm/operations.rst [hunk 3]
```diff
@@ -254,3 +254,24 @@ You can remove a broken host from management with::
...
+
+In order to set initial configuration settings befor calling
```
Comment (mgfritch) [inline doc/cephadm/operations.rst:270]:
```suggestion
To set an initial configuration before calling 
```

File: doc/cephadm/operations.rst [hunk 4]
```diff
@@ -254,3 +254,23 @@ You can remove a broken host from management with::
 You can disable this health warning with::
 
   ceph config set mgr mgr/cephadm/warn_on_failed_host_check false
+
+/etc/ceph/ceph.conf
+===================
+
+Cephadm uses a minimal ``ceph.conf`` only containing the 
+minimal set of information to connect to the Ceph cluster.
```
Comment (asettle) [inline doc/cephadm/operations.rst:262]:
```suggestion
Cephadm uses a minimized ``ceph.conf`` that only contains 
a minimal set of information to connect to the Ceph cluster.

File: doc/cephadm/operations.rst [hunk 5]
```diff
@@ -254,3 +254,23 @@ You can remove a broken host from management with::
...
+
+To update the configuration settings, please use::
```
Comment (asettle) [inline doc/cephadm/operations.rst:264]:
```suggestion
To update the configuration settings, use::

File: doc/cephadm/operations.rst [hunk 6]
```diff
@@ -254,3 +254,23 @@ You can remove a broken host from management with::
...
+
+  ceph config set ...
+
+
+To set an initial configuration before calling 
+bootstrap, please create an initial ``ceph.conf`` like so::
```
Comment (asettle) [inline doc/cephadm/operations.rst:270]:
```suggestion
To set up an initial configuration before calling
`bootstrap`, create an initial ``ceph.conf`` file. For example::

**Comment by sebastian-philipp** [comment]
thanks for the feedback!

**Pull request merged**